Sources/JSONPatch/JSONPatch.swift

```diff
@@ -28,6 +28,13 @@ import Foundation
 /// https://tools.ietf.org/html/rfc6902
 public class JSONPatch: Codable {
 
+    /**
+     Default setting for ignoring non existent values (Default: false)
+     
+     When set to true, all JSONError.referencesNonexistentValue errors will be ignored
+     */
+    public static var ignoreNonexistentValues: Bool = false
```

Comment:
That flag wasn't meant to be toggled on and off during the app's runtime..
Merely to just set once (e.g. in AppDelegate).. 
e.g. it's set to `false` by default, to maintain backward compatibility. I would set it to true in AppDelegate cause it would be my default for the whole app's life cycle.

But true, if you're going to use that option set, overriding the default set would work as well